examples/azure/terraform-azure-appserviceplan-example/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "resourceGroupName" {
```

Comment:
NIT:

- Need the standard variable blocks that break up into the sections Env Vars, Required Vars, and Optional Vars.
- The standard style order of variables we use is `description`, `type`, `default` in that order.